### ISSUE TITLE
We don't ensure start up services are up before an uninstall

### ISSUE THREAD

**Issue by pirog** [kalabox/kalabox#260]
```
kbox uninstall -- --purge
info: DOCKER =>  Removing container 7a7b35a3b72285e3c68bd50a5135a433ad7e3de6cd0cf518886d038c6b70c102. v=true, kill=false
info: DOCKER =>  Container removed. 7a7b35a3b72285e3c68bd50a5135a433ad7e3de6cd0cf518886d038c6b70c102
Removing the codez.

info: DOCKER =>  Removing container fdbd1ee56c6b96962ab0d2a7bc5b7f975311521346fd85c300f5444694b90f08. v=true, kill=false
info: DOCKER =>  Container removed. fdbd1ee56c6b96962ab0d2a7bc5b7f975311521346fd85c300f5444694b90f08
Redis connection to 10.13.37.42:8160 failed - connect ECONNREFUSED
```

**Comment by pirog** [comment]
So i see two obvious pathways here @bcauldwell 

1. Make sure start up services are on before we do this
2. Don't worry about removing the entry from redis to begin with

thoughts?

**Comment by bcauldwell** [comment]
I'm torn.

On one hand it seems a little crazy to make sure startup services are
running before uninstalled an app.

Now also keep in mind we didn't remove redis entries until I fixed the code
to do so in 0.8.0, so we've lived very long without it and it didn't even
seem to cause a problem.

We could also try to remove the entry from redis and then just fail
silently if redis can't connect.

I think I need to think about this a little more, but should definitely be
treated as a bug and should get fixed ASAP.

On Mon, Jun 29, 2015 at 4:45 PM, Mike Pirog <notifications@github.com>
wrote:

> So i see two obvious pathways here @bcauldwell
> <https://github.com/bcauldwell>
>
>    1. Make sure start up services are on before we do this
>    2. Don't worry about removing the entry from redis to begin with
>
> thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/260#issuecomment-116880678>.
>



-- 
- Benjamin Cauldwell
